models/token.js

```diff
@@ -54,7 +61,7 @@ module.exports = {
      * @property base
      * @type {Joi}
      */
-    base: Joi.object(MODEL).label('Token'),
+    base: Joi.object(MODEL).without('userId', 'pipelineId').label('Token'),
```

Comment:
It passed when both `userid` and `pipelienId` are absent. Is this ok? If not, how about using [xor] (https://github.com/hapijs/joi/blob/7a0582d1b7b473b5d7e3f7f4e97beaf918f1229c/lib/types/object/index.js#L484).

Reply:
Originally, `userId` is not required, so I can't decide it should be required.
https://github.com/sonic-screwdriver-cd/data-schema/blob/5979c9237bcf6c8fdf56418576c3d30e05f7f9eb/models/token.js#L24

Reply:
🙆